app/javascript/v3/components/Form/Textarea.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <with-label
+    :label="label"
+    :name="name"
+    :has-error="hasError"
+    :error-message="errorMessage"
+  >
+    <template #label> {{ label }} <slot /> </template>
```

Comment:
Do we need this, since the label is already passed, and we are not doing anything custom with the slot